audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/AuditApplication.java

```diff
@@ -23,15 +23,22 @@
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
 
-import java.util.Arrays;
-
+/**
+ * SpringBoot application entry-point class for audit-service executable
+ * All spring components to be scanned must be in this package
+ */
 @EnableEurekaClient
 @SpringBootApplication
 public class AuditApplication {
     private static final Logger LOGGER = LoggerFactory.getLogger(AuditApplication.class);
 
+    /**
+     * SpringBoot application entry-point method for audit-service executable
+     *
+     * @param args command-line arguments passed to the application
+     */
     public static void main(final String[] args) {
-        LOGGER.debug("AuditApplication started with: {}", AuditApplication.class.toString(), "main", Arrays.toString(args));
+        LOGGER.debug("AuditApplication started with: {}", (Object) args);
```

Comment:
I don't understand why you have chosen to cast to Object rather than just using Arrays.toString(args)

Reply:
Conditional evaluation of the expression:
* If debug logging is disabled, then time is wasted evaluating the Arrays.toString when it won't be used.
* If debug logging is enabled, then a String[] args can be treated as a varargs, so explicitly cast to Object